src/qudi/hardware/interfuse/ni_scanning_probe_interfuse.py

```diff
@@ -520,50 +519,40 @@ def _check_scan_end_reached(self):
         if self.__scan_stopped:
             return True
 
-        if self._scan_data.scan_dimension == 1:
+        if len(self.raw_data_container) == self._ni_finite_sampling_io().frame_size:
```

Comment:
If we stick with the RawData structure in it's own class: Maybe a .is_full() property could be neat...

Reply:
With feedback from @Neverhorst, probably the "RawData" structure will be dropped. But we can also discuss again today.

Reply:
Did introduce the requested property in 97451e42. Even though the RawDataContainer will not last long after this PR.